components/script/dom/bindings/root.rs

```diff
@@ -721,6 +721,16 @@ impl<T: DomObject> MallocSizeOf for MutNullableDom<T> {
     }
 }
 
+impl<T: DomObject> Clone for MutNullableDom<T> {
+    #[allow(crown::unrooted_must_root)]
+    fn clone(&self) -> Self {
+        let inner = unsafe { &*self.ptr.get() };
+        Self {
+            ptr: UnsafeCell::new(inner.clone()),
+        }
+    }
+}
```

Comment:
I think there is a problem with this `Clone`, because I have checked what `branch_1` is set to, and even though one clone is some after having been set, the other one held by the tee source is still none.